code/game/turfs/exterior/_exterior.dm

```diff
@@ -18,8 +18,6 @@
 	var/icon_edge_layer = -1
 	var/icon_edge_states
 	var/icon_has_corners = FALSE
-	///If this turf is on a level that belongs to a planetoid, this is a reference to that planetoid.
-	var/datum/planetoid_data/owner
```

Comment:
Surprised this was okay, but it really is only used in exterior's `Initialize()` and in `SSmaterials.get_rock_color()`.